modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -213,6 +245,74 @@ const getDriverSpec = (settings: DriverSettings, browserName: string): DriverLoa
   };
 };
 
+const driverSetup = async (driver: WebdriverIO.Browser, settings: DriverSettings, debuggingPort: number): Promise<void> => {
+  // Browsers have a habit of reporting via the webdriver that they're ready before they are (particularly FireFox).
+
+  // setTimeout is a temporary solution, VAN-66 has been logged to investigate properly
+  await driver.pause(1500);
+
+  // Log driver details
+  logDriverDetails(driver, settings.headless, debuggingPort);
+
+  // Some tests require large windows, so make it as large as it can be.
+  // Headless modes can't use maximize, so just set the dimensions to 1280x1024
+  if (settings.headless) {
+    await driver.setWindowSize(1280, 1024);
+  } else {
+    await driver.maximizeWindow();
+  }
+  
+  return Promise.resolve();
+};
+
+const getFarmUrl = async (): Promise<URL> => {
+  console.log('Creating DeviceFarmClient...');
+  const client = new DeviceFarmClient({region: 'us-west-2'});
```

Comment:
Should the region here be configurable as a setting or be pulled from env?

Reply:
Variables are taken as args or read from env as suggested